docs/config/index.md

```diff
@@ -879,6 +879,32 @@ Pass additional arguments to `node` process in the VM context. See [Command-line
 Be careful when using, it as some options may crash worker, e.g. --prof, --title. See https://github.com/nodejs/node/issues/41103.
 :::
 
+### parallelism
+
+- **Type:** `boolean`
+- **Default:** `true`
+- **Version:** Since Vitest 1.1
+
+Should all test files run in parallel. This will override `maxWorkers` and `minWorkers` options.
```

Comment:
I wonder if it we should add the override value that will be used...

```ts
When set to `false`, this will override `maxWorkers` and `minWorkers` options and use value `1`.
```